src/pages/HomePage/pages/MarketPlace/MarketPlace.tsx

```diff
@@ -41,8 +41,7 @@ export function MarketPlace() {
     loading: isUpdating,
   } = usePut(api.put.updateMarket);
   const { executeDelete, success } = useDelete(api.delete.deleteMarket);
-    const { data: events } = useFetch(api.fetch.fetchAllUniqueEvents);
-
+  const { data: events } = useFetch(api.fetch.fetchAllUniqueEvents);
```

Comment:
nope please use data stored in the store it is reactive and update changes in real time

i realised the endpoint has been changed but still use that because that gives as a single source of truth once i correct it it will affect every place instead of hunting for all use cases